src/SonarQube.Client/Models/Connections.cs

```diff
@@ -110,4 +114,12 @@ public void Dispose()
         }
         #endregion
     }
+
+    public class ConnectionDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class 'ConnectionDTO' to match camel case naming rules, consider using 'ConnectionDto'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS101)